src/oauth2/index.ts

```diff
@@ -162,12 +162,12 @@ export class OAuth2RequestError extends Error {
 	}
 }
 
-interface TokenErrorResponseBody {
+export interface TokenErrorResponseBody {
```

Comment:
I believe this is an internal type in that there are no public APIs or generics that use it

Reply:
Good point. I've removed the export. At first I had considered the interface could also be useful externally, but you're right it makes less sense if it is not actually part of the public API.